source/extensions/transport_sockets/alts/BUILD

```diff
@@ -34,6 +34,7 @@ envoy_cc_extension(
     hdrs = [
         "config.h",
     ],
+    category = "SOMECAT",
```

Comment:
i see 

```
$ grep transport_socket catlist.txt 
envoy.transport_sockets.downstream
envoy.transport_sockets.upstream
```
and not sure which is correct

Reply:
This can apply to both; we should probably allow `category` to be a list.

Reply:
hmm, i wondered about this

Reply:
i guess we can overload the param in `envoy_cc_extension`